meta-efi-secure-boot/recipes-bsp/grub/grub-efi-efi-secure-boot.inc

```diff
@@ -89,7 +89,7 @@ EOF
     > ${WORKDIR}/cfg
 	fi
 	cat<<EOF>>${WORKDIR}/cfg
-search.file ${GRUB_PREFIX_DIR}/grub.cfg root
+search.file (\$cmdpath)${GRUB_PREFIX_DIR}/grub.cfg root
```

Comment:
Please help to fix another issue here:
```
search.file (\$cmdpath)${GRUB_PREFIX_DIR}/grub.cfg
```

According to https://www.gnu.org/software/grub/manual/grub/html_node/search.html, the default variable used to store the found device is 'root'. In addition, the syntax of search.file command doesn't't support the second parameter.